handout/testing/components/injecting-dependencies.md

```diff
@@ -83,12 +94,18 @@ describe('Testing Quote Component', () => {
 ```
 [View Example](http://plnkr.co/edit/7KZu1Yg6kBX7rksrpRHV?p=preview)
 
-Testing the `QuoteComponent` is a fairly straightforward process. We want to create a `QuoteComponent`, feed it a quote and see if it appears in the DOM. This process requires us to create the component, pass in any dependencies, trigger the component to perform an action and then look at the DOM to see if the action is what we expected. Let's take a look at how this is accomplished with the above unit test.
+Testing the `QuoteComponent` is a fairly straightforward process. We want to create a `QuoteComponent`, feed it a quote and see if it appears in the DOM. This process requires us to create the component, pass in any dependencies, trigger the component to perform an action and then look at the DOM to see if the action is what we expected. 
+
+Let's take a look at how this is accomplished with the above unit test.
+
+We use `TestBed.initTestingEnvironment` to create a testing platform using `BrowserDynamicTestingModule` and `platformBrowserDynamicTesting` as arguments, which are also imported from angular and allow the application to be bootstrapped for testing. This is necessary for all unit tests that make use of `TestBed`. Notice that this platform is destroyed and reset before each test runs.
```

Comment:
Yeah, my knowledge here is shaky at best. In the starter repo we kind of obscure all of this with a configuration file. What I've documented is kind of where I came out after playing with it on a per-test basis and asking around, but it doesn't quite line up with how it's apparently supposed to work.